### ISSUE TITLE
Dropout

### ISSUE THREAD

**Issue by ameliajimenez** [taehoonlee/tensornets#42]
Hello!

Thank you for sharing this library, it's great work!

I would like to know whether it's possible to use dropout with the pretrained models, I'm obtaining severe overfitting in my models. 

Thanks in advance.

Regards,
Amelia.

**Comment by taehoonlee** [comment]
Thank you for trying TensorNets, @ameliajimenez. Currently, you need to modify the model in your local if you want to customize the model.
